contracts/src/Creator.sol

```diff
@@ -3,8 +3,37 @@ pragma solidity 0.7.1;
 
 import "buidler-deploy/solc_0.7/proxy/Proxied.sol";
 import "@nomiclabs/buidler/console.sol";
+import "@openzeppelin/contracts/token/ERC1155/ERC1155PresetMinterPauser.sol";
+import {
+    ISuperfluid,
+    ISuperToken,
+    ISuperAgreement,
+    ISuperApp,
+    SuperAppDefinitions
+} from "@superfluid-finance/ethereum-contracts/contracts/interfaces/superfluid/ISuperfluid.sol";
+
+import {
+    IConstantFlowAgreementV1
+} from "@superfluid-finance/ethereum-contracts/contracts/interfaces/agreements/IConstantFlowAgreementV1.sol";
+
+contract Creator is ERC1155PresetMinterPauser, ISuperApp {
+
+    /// @dev Collateral fee to subscribe, will stream back over a month (hardcoded to $5)
+    uint256 constant private _COLLATERAL_FEE = 1e18;
+    /// @dev Minimum flow rate to subscribe (hardcoded to $5 / mo)
+    int96 constant private _MINIMUM_FLOW_RATE = int96(uint256(5e18) / uint256(3600 * 24 * 30));
```

Comment:
I'm guessing it'd be better to hard-code the result of the operation and put the actual math in a comment to save on gas costs